bundles/org.openhab.binding.energidataservice/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -26,6 +26,10 @@
 		<label>CO₂ Emission</label>
 		<description>CO₂ emission in g/kWh.</description>
 		<category>Smoke</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>CO2</tag>
```

Comment:
Please remove this. The channel types is used for two different channels: One is a prognosis, and the other one is "real-time", but I think in both cases they are calculated, not measured. See:

https://github.com/openhab/openhab-addons/blob/e4bc05a1e1e5d1f5bc5a19323114340a3cca7bd4/bundles/org.openhab.binding.energidataservice/src/main/resources/OH-INF/thing/channel-groups.xml#L35-L42

Reply:
Shall we use the (not yet merged) `Calculation` tag?

Reply:
I don't think calculation is really representative. Of course calculation is part of it, at least adding numbers from different sources, I'd assume. I don't really know how the numbers are created, but I'd assume the different power plants will report numbers, and an average for the region is then created from that. But don't really know. I'd still propose to remove them, since it seems fuzzy.

Reply:
Removed